src/gossip/gossip_service.zig

```diff
@@ -1528,6 +1543,37 @@ pub const GossipService = struct {
         }
     }
 
+    /// Attempts to associate each entrypoint address with a contact info.
+    /// Returns true if all entrypoints have been identified
+    fn identifyEntrypoints(self: *Self) bool {
```

Comment:
rename to `populateEntrypointsFromCrdsTable`? since thats what were doing?

Reply:
https://github.com/Syndica/sig/pull/69/commits/e4fdca479e235e29a5df018ea2ff9c7275dd17c4